docs/content/preview/drivers-orms/_index.md

```diff
@@ -13,16 +13,14 @@ menu:
     parent: develop
     weight: 570
 type: indexpage
+showRightNav: true
 ---
 
 The [Yugabyte Structured Query Language (YSQL) API](../api/ysql/) builds upon and extends a fork of the query layer from PostgreSQL, with the intent of supporting most PostgreSQL functionality. Client applications can use the [PostgreSQL drivers](https://www.postgresql.org/download/products/2-drivers-and-interfaces/) to read and write data into YugabyteDB databases. YSQL-compatible PostgreSQL drivers are listed in the compatibility matrix below.
 
-In addition to the compatible upstream PostgreSQL drivers, YugabyteDB also supports **smart drivers**, which extend the PostgreSQL drivers to enable client applications to connect to YugabyteDB clusters without the need for external load balancers. YugabyteDB smart drivers have the following features:
+## Smart drivers
```

Comment:
Change the driver titles wherever applicable on this page-
YugabyteDB PGX, Yugabyte psycopg2 and any others.